web/react/components/textbox.jsx

```diff
@@ -303,7 +303,19 @@ export default class Textbox extends React.Component {
     }
 
     render() {
-        const previewLinkVisible = this.props.messageText.length > 0;
+        let previewLink = null;
+        if (Utils.featureToggle('#1389')) {
```

Comment:
`'#1389'` is supposed to be the name of the feature? This should be moved to a constant with a better name.